ci/kind/test-e2e-kind.sh

```diff
@@ -99,7 +99,7 @@ if $coverage; then
     manifest_args="$manifest_args --coverage"
     COMMON_IMAGES_LIST+=("antrea/antrea-ubuntu-coverage:latest")
 else
-    COMMON_IMAGES_LIST+=("antrea/antrea-ubuntu:latest")
```

Comment:
Do you have any good way to define the registry URL once in a file and other code just read the URL?

Reply:
Or maybe have an `update-image-name-with-registry.sh` script to update all images' names in the repo? Then we don't need to add logic in the code.

Reply:
I guess specifically for test / CI, we can have something like this:

images.yml
```yaml
antrea/antrea-ubuntu: projects.registry.vmware.com/antrea/antrea-ubuntu
```

get_img.sh
```bash
function get_img {
    while read -r line; do
        IFS=':' read -ra ADDR <<< "$line"
        name=$(echo ${ADDR[0]} | xargs)
        img=$(echo ${ADDR[1]} | xargs)
        if [[ $name == $1 ]]; then echo $img; fi
    done < "images.yml"
}

[[ $_ != $0 ]] || get_img $1
```

for example, CI scripts can source `get_img.sh` and use `"$(get_img antrea/antrea-ubuntu)"`

I am not sure it's much better though. Maybe we can wait a bit and see how things evolve in the short term? Right now we have to push and pull from different registries which is making things a bit tricky.

Reply:
Good idea. Yes we can wait and see.